main-repo-triage/checkissue.py

```diff
@@ -15,27 +15,48 @@
 with open('environment_titles.json', 'r', encoding='utf8') as envtitlesfile:
     env_titles = json.load(envtitlesfile)
 
-def checkissue(i:github.Issue.Issue):
+
+def checkissue(i: github.Issue.Issue):
```

Comment:
might as well
```suggestion
def checkissue(i: github.Issue.Issue) -> Optional[str]:
```
or something
while you're at it